### PR TITLE
add new question(closure)

### PR DESCRIPTION
**Pull request by Piyush-zlatan**

### CODE REVIEW

**Review by arpansac — CHANGES_REQUESTED** [review]
Everything seems fine, you just need to resolve a conflict because someone added a question already. New thing to learn!

**Comment by Piyush-zlatan** [comment]
> Everything seems fine, you just need to resolve a conflict because someone added a question already. New thing to learn!

resolved..!! please check

File: javascript/answers/q4.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        gtag('config', 'UA-157837769-1');
+        </script>
+        
+        <title>
+            JavaScript | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content="What will be the output of the code below?">
```
Comment (arpansac) [inline javascript/answers/q4.html]: Please change the content="" part to be the text of the question

File: javascript/answers/q4.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            <img src="../../assets/CN_logo.png" height='72px' alt="Coding Ninjas Logo">
+        </div>
+
+        <div>
+            <p>
+                <small>
+
+                    <a href="../../index.html">Home</a>
+                    &gt; <a href="../index.html">JavaScript Questions</a>
+                    &gt; What will be the output of the code below?
```
Comment (arpansac) [inline javascript/answers/q4.html]: Change this to the question text

**Review by arpansac — CHANGES_REQUESTED** [review]

**Review by arpansac — APPROVED** [review]

**Pull request merged**